### PR TITLE
Batch create users

### PR DESCRIPTION
**Pull request by aarturoc**
This PR contains process of batch create users

[Preview Diagram](https://github.com/GalileiLearning/glowing-octo-dollop/tree/batch-create-users/7)

### CODE REVIEW

File: 7/README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-# Batch create users
\ No newline at end of file
+# Batch create users
+As an admin, I want to add a list of new user so they can log in the aplication.
```
Comment (andymotagl) [inline 7/README.md:2]: What do you think about _Logging in to the system_?

File: 7/README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+### Actors
+Admin
+
+### Preconditions
+The admin needs a list of users corporative emails
```
Comment (andymotagl) [inline 7/README.md:8]: Be authenticated in the platform?

File: 7/README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+### Scenarios
+#### Success
+
+Admin add the list of new users.
```
Comment (andymotagl) [inline 7/README.md:13]: nit: adds

File: 7/README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+#### Failure
+
+1. The input information is not correct.
```
Comment (andymotagl) [inline 7/README.md:17]: Would it be considered incorrect response when some are correct but others no?

File: 7/README.md [hunk 5]
```diff
@@ -1 +1,36 @@
...
+2. The user already exist.
+
+### Flow Diagram
+
+```mermaid
+flowchart TD
+    A[Start] --> M[Take one user of the list]
+    M --> B{The infomration is correct?}
```
Comment (andymotagl) [inline 7/README.md:25]: Typo: information

**Comment by aarturoc** [comment]
Closing this cause i will make other PR with the changes

**Pull request closed without merging**